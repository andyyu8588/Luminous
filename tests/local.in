// For compiler/VM testing purpose

{
  a = 1;
  b = "poggers";
  {
    a = 2;
    b = 1 < 2;
    print(a);
    print(b);
  }
  print(a);
  print(b);
}