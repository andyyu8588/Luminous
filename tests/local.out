2
true
1
poggers
